gen.conf

```diff
@@ -0,0 +1,43 @@
+# zap-baseline rule configuration file
+# Change WARN to IGNORE to ignore rule or FAIL to fail if rule matches
+# Only the rule identifiers are used - the names are just for info
+# You can add your own messages to each rule by appending them after a tab on each line.
```

Comment:
I think a lot of these are Drupal specific and it would be better to "fail safe" and have all rules warn (or even fail, at least for serious issues) by default. Having these here means that static HTML or other frameworks could miss issues that they need to fix.
For the Drupal specific rules, I think we could provide example file that projects could copy into their tests/zap dir and customize. For that we would probably want to be pretty conservative also (i.e. only ignore rules that a fully hardened and carefully configured best case Drupal site would still fail due to the architectural limitations etc).
For the Drupal example, it's good to include comments for any rule ignored to describe why it is safe to do so etc.

Reply:
How can I tell if a rule is "Drupal-specific"? I don't understand the scope of applications that these rules could apply to, so these rules look to me like they could apply to a wide range of apps. I see what you mean about anything involving cookies/sessions not applying to static sites, but won't those tests just pass?

Are there any references or sources with better sets of defaults, or at least lists of rules that are/aren't for different apps so I have an idea of what to change?

Reply:
Realized it was the bottom code. Reset this to the baseline you get with `zap -g gen.conf` so there's no more Drupal things.